### DISCUSSION TITLE
SHOWCASE

### DISCUSSION THREAD

**Discussion by TT-ReBORN** [TT-ReBORN/Georgia-ReBORN#86] [Show and tell]
Hello community,

I've seen some pretty cool pictures of Georgia-ReBORN and I think it would be cool to make a dedicated thread for a showcase.
Go ahead and post as many pictures as you like, let me see your best designs! Other users will be also interested, have fun!

It would be also helpful for others to post which theme, theme styles or theme style preset you were using!

-TT

**Comment by d3rb** [comment]
![Placeholder1](https://user-images.githubusercontent.com/15198548/199318823-c5c26591-59d6-4568-b6cf-a95a5d606d72.png)
![Placeholder2](https://user-images.githubusercontent.com/15198548/199318881-033c4526-ebe3-4067-b9b8-8dcad0e3319f.jpg)

**Comment by Zephyr0ck** [comment]
Mine.

```
- Georgia-ReBORN v2.3.0 Beta 10b
- foobar2000 v2.0 beta 12 : 32-bit
```

![Capture_04](https://user-images.githubusercontent.com/66889502/200187660-e8ba1db1-daf0-4e99-b6a1-bef7b68e6b8d.PNG)
![Capture_05](https://user-images.githubusercontent.com/66889502/200187662-2d1e5246-2aa9-4845-a7dd-772fcc16a3b1.PNG)
![Capture_03](https://user-images.githubusercontent.com/66889502/200187656-2260d739-3c8d-4257-82bd-2e5eac1f24e6.PNG)
![Capture_01](https://user-images.githubusercontent.com/66889502/200187651-e4f4d717-29e6-4362-b513-c4ab55e56078.PNG)
![Capture_02](https://user-images.githubusercontent.com/66889502/200187654-4559b273-bf57-4652-ac19-e605293663a8.PNG)

**Reply by TT-ReBORN** [reply]
Looks great, José =)

What fonts you're using? I think people want to know.

**Comment by d3rb** [comment]
![Bandmaschiene1](https://user-images.githubusercontent.com/15198548/201523214-04829995-59fc-49d8-9175-58653e298e0b.png)

**Reply by TT-ReBORN** [reply]
Haha, looks pretty cool and silly ;-)

**Comment by Zephyr0ck** [comment]
```
foobar2000 v2.0 beta 13 : 32-bit
Georgia-ReBORN v2.3.0 Beta 10d
```

I've created two new Vinyl Disc art for placeholders 😋 
![Georgia-ReBORN_beta10d_Wednesday, November 23, 2022_10h3m29s_001_Rosenrot - foobar2000](https://user-images.githubusercontent.com/66889502/203595218-855f97d1-4af4-4a7a-b63c-8c84cd5fdc20.png)
![Georgia-ReBORN_beta10d_Wednesday, November 23, 2022_10h4m34s_002_Ohne Dich - foobar2000](https://user-images.githubusercontent.com/66889502/203595229-968e04ad-f7df-4278-9389-3c4e6c6ab540.png)
![Georgia-ReBORN_beta10d_Wednesday, November 23, 2022_10h6m25s_003_Ohne Dich - foobar2000](https://user-images.githubusercontent.com/66889502/203595234-069ce83b-d3b1-49e3-984d-fc49531c84c8.png)
![Georgia-ReBORN_beta10d_Wednesday, November 23, 2022_10h6m50s_004_Ohne Dich - foobar2000](https://user-images.githubusercontent.com/66889502/203595239-9ad881df-f8af-46d6-859c-310b1e794a44.png)
![Georgia-ReBORN_beta10d_Wednesday, November 23, 2022_10h7m1s_005_Ohne Dich - foobar2000](https://user-images.githubusercontent.com/66889502/203595244-73aac0eb-f6ae-4304-9626-050385938600.png)
![Georgia-ReBORN_beta10d_Wednesday, November 23, 2022_10h8m54s_001_Mann Gegen Mann - foobar2000](https://user-images.githubusercontent.com/66889502/203595252-64e3c21f-9f49-4f38-9dec-e290fb13884c.png)

Thanks Tom, you did a fantastic and awesome work with Georgia-ReBORN! 👍

**Reply by TT-ReBORN** [reply]
Thanks, your screenshots looking clean and smooth as always. 
The new vinyl art placeholders are in sync with the designs, so good job =)

**Comment by Zephyr0ck** [comment]
Hi, it's me again.

```

- foobar2000 v2.0 beta 33 @ 32-bit
- Georgia-ReBORN v2.3.0 FINAL Beta 12
- Theme/Style → Bevel → Blend 2 → Reborn fusion accent
- Custom Vinyl place holders.

```
Peakmeter bar and waveform are great, but for me old-school GRB it's dope...

Thanks @TT-ReBORN for your hard work and all your time dedicated to improve and polish to the best foobar2000 theme!

Vinyl Oxblood:
![2023-04-16_142717](https://user-images.githubusercontent.com/66889502/232606641-20cbc2c5-ca99-40bf-a072-c4d344084b25.png)
Vinyl Georgia-ReBORN:
![2023-04-16_142908](https://user-images.githubusercontent.com/66889502/232606650-62caed5c-2ebf-441b-b44f-0a59e61fa5f9.png)
Vinyl White/Smoke:
![2023-04-16_143049](https://user-images.githubusercontent.com/66889502/232606657-436bbb46-7f69-4be7-b96a-0ea9c59193e8.png)
Vinyl Purple/Pink:
![2023-04-16_143141](https://user-images.githubusercontent.com/66889502/232606664-57062c5c-f3e7-4b1a-8edc-424db4611c9d.png)
Vinyl Mint/Green:
![2023-04-16_143246](https://user-images.githubusercontent.com/66889502/232606673-c404aab6-7a4e-4136-b695-97f4c18e4383.png)
Vinyl Magenta:
![2023-04-16_143338](https://user-images.githubusercontent.com/66889502/232606677-c52f5be4-938c-41c4-ba46-bfa3cbf86d7a.png)
Vinyl Sky/Blue:
![2023-04-16_143558](https://user-images.githubusercontent.com/66889502/232606680-1f9c46a5-33a6-48ba-b01e-dfcc13a2b5f9.png)
Vinyl Goldess:
![2023-04-16_143709](https://user-images.githubusercontent.com/66889502/232606684-a32e54cf-f18d-43ca-99d5-cc994fbd69cd.png)
Vinyl White/Pearl:
![2023-04-16_143809](https://user-images.githubusercontent.com/66889502/232606688-26407463-0229-4659-84ee-0302acb09537.png)
Vinyl Red:
![2023-04-16_143916](https://user-images.githubusercontent.com/66889502/232606695-20ba5e4e-4671-464d-850e-5262ac4bc5f5.png)
Vinyl Blue:
![2023-04-16_144039](https://user-images.githubusercontent.com/66889502/232606711-61bcebdd-d415-47e8-9e48-4e622139f1b0.png)
Vinyl Clear/Smoke:
![2023-04-16_144307](https://user-images.githubusercontent.com/66889502/232606715-5be18806-1def-4bfa-82d3-9adf38576b10.png)
Vinyl Red/Black Smoke:
![2023-04-16_152604](https://user-images.githubusercontent.com/66889502/232606730-a605c152-f6ab-4b2a-9dae-d59f5d391592.png)
Vinyl Black Smoke:
![2023-04-16_152718](https://user-images.githubusercontent.com/66889502/232606756-1edab9b4-5203-4591-8812-c0f2e6ec24d7.png)
Library Tree:
![2023-04-16_152848](https://user-images.githubusercontent.com/66889502/232606773-93b41e55-23f0-423a-b8d4-cd45938da0a2.png)
Biography:
![2023-04-16_152933](https://user-images.githubusercontent.com/66889502/232606796-bb11945b-7c14-461c-9326-e7d8a59b32d7.png)
Lyrics:
![2023-04-16_155148](https://user-images.githubusercontent.com/66889502/232606629-ae02a520-026b-46d6-a55d-18dc486ab532.png)

**Reply by TT-ReBORN** [reply]
Hi José! Long time no see, I thought you were kidnapped or something =)

Glad you like the new stuff, what do you think of the new Reborn fusion ( extra additional color from album artwork )?
**Btw, please post as many pictures as you like here, this was the purpose for this thread, it's no spam but good promotion!**
It's much better to directly embed and show the pictures in your posts, instead of posting external links!
If it's not much work for you, you can edit your post and embed them. You always make very nice screenshots!

Thanks! 
Tom

**Reply by giorgiobrit** [reply]
awesome !!!!!!

**Reply by Zephyr0ck** [reply]
> Hi José! Long time no see, I thought you were kidnapped or something =)
> 
> Glad you like the new stuff, what do you think of the new Reborn fusion ( extra additional color from album artwork )? **Btw, please post as many pictures as you like here, this was the purpose for this thread, it's no spam but good promotion!** It's much better to directly embed and show the pictures in your posts, instead of posting external links! If it's not much work for you, you can edit your post and embed them. You always make very nice screenshots!
> 
> Thanks! Tom

I was kidnapped by my job :P 

Reborn fusion fills every corner in Georgia REBORN. For me right now Georgia ReBORN doesn't need anything else.

Done, I've added all my pictures into the original post. 

btw. I'm planning to release my Vinyl placeholders for everyone.

**Reply by TT-ReBORN** [reply]
Great! [Everyone can post them here, preview pics would be also helpful](https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/85).
Unfortunetaly you can't upload the zip into the post, 25MB is the maximum size limit in Github posts. 
I would recommend to upload it on some free space provider and include the link to the download in your post.
-Tom

**Reply by fwn0** [reply]
Nice work Zephyr0ck, can you save ( Options > Theme > Custom > Save current colors ) Vinyl Black Smoke, Vinyl White/Pearl, Vinyl Mint/Green and Vinyl Oxblood to your georgia-reborn-custom-config and share it here? It could be nice start for my own config ;)

**Comment by acstrp** [comment]
Just the standard Reborn theme + Bevel but this is my favorite album so far for scroll wheel cover switching.

It was originally released with 4 alternate covers - not sure if the fifth one is accurate but I included it in the set anyway:

![animated](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/36980498/f070c2c6-de22-4f98-9027-d237fd934cc2)

**Reply by TT-ReBORN** [reply]
Yea it's cool, I only saw this with booklets =) . 
Stay tuned for the harmonic mode in the upcoming version, I really like it.

**Comment by karehaqt** [comment]
New version installed and looking great, just using Black, Blend 1 and Alternative 2

![8QB1gIY](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/8368361/1a20e7cf-4b2c-4423-8fa4-7e8abf3fac90)
![4b5rKth](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/8368361/b88e35f1-e01d-47bd-96fe-4f2afb369416)
![OdBJtxh](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/8368361/2fd75f94-5002-483d-8ad2-1b8a59249199)

**Reply by TT-ReBORN** [reply]
Looks clean with the black CD placeholder, it was a good idea that I designed those =)
I have lots of great ideas and designs for some other new placeholders that will simulate cool animation effects.

**Comment by d3rb** [comment]
I Love my Sony Highspeed Reel 2 Reel Tape and i make a new Placeholder. 
I know it's not to everyone's taste but I like it.
greetings

![IMG20230818131742](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/15198548/f0b84c4d-9b16-4ac1-b5e1-deb2089ebbe6)

https://github.com/TT-ReBORN/Georgia-ReBORN/assets/15198548/268b3e65-de3a-4870-a7d9-7a6aacb4ae52

**Reply by TT-ReBORN** [reply]
That looks really cool, you should make zip and release it here.

When I have more free time I will design some new placeholders too, with some cool special effects.
I got some nice and crazy ideas =)

**Comment by d3rb** [comment]
ok now i will publish my placeholder here.

Tom once inspired me to do this.
He helped me when I had a problem,
I was so fascinated by his work, 
that I thought I'd build a placeholder as a thank you.

I started a long time ago with one and now 18 placeholders have emerged over time

Attention! Important!!

All vinyls and tape reels are my own property.
They were photographed and edited in Photoshop.
I've never used Photoshop before,
so some may not find my placeholders perfect,
but they do for me, and I've learned a lot from it.

Thank you Tom and everyone else for your hard work!!
and sorry for my English i do my best. :) 

Enrico

![Publish Placeholder](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/15198548/2c9fc601-027d-4605-8173-3beace5dfd21)
![Publish Vinyl Placeholder](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/15198548/53c303cd-cdc9-408e-af2d-47b6ad292295)

Download: https://github.com/d3rb/Placeholder-for-Georgia-ReBORN/tree/main
------------
edit 03.09.2023:
fix.centered.NAB-Adapter for all Reels + add some New Reels

https://github.com/d3rb/Placeholder-for-Georgia-ReBORN/blob/main/FIXED.Reel2Reel.Centered.NAB.rar

**Reply by TT-ReBORN** [reply]
That is great start, thanks for sharing =)
I would like and love to see more ( custom themes, placeholders ) from the Georgia-ReBORN community.

**Comment by d3rb** [comment]
Hello guys, here are a few new Reel2Reel placeholders in airbrush style, for everyone who looking for more color and unusual placeholders!!! All placeholders are in high resolution..

have fun with it.
With kind regards
d3rb

https://github.com/d3rb/Placeholder-for-Georgia-ReBORN/blob/main/Reel2Reel.Airbrushstyled.Colored.NAB.rar

![Foobar Theme Orange](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/15198548/77896b67-e899-4fb8-9651-99d525d3f5ba)
![R2R Planet 1 Logo+NAB-Big New Ring Shadowed](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/15198548/c2b36fa0-3846-4aa1-b57a-9679093b47d2)
![New Airbrushed Reels Wallpaper](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/15198548/fa347ab6-f247-4997-b27c-cc1ef1e6ce8d)

**Reply by TT-ReBORN** [reply]
Great work Enrico! I really like these =)
Would be really cool if more people like you would create these and share them for the community!

**Reply by d3rb** [reply]
Hello Tom, nice to hear that you liked it, i hope others too.

Is it possible to create an edit button or a solution to add own or more names for placeholders?


Another question, we have two options to rate a music track: once via the library by clicking the "points" and once via the top menu (Bar) “Rating”.
I always find it difficult to rate the album via right click -> Playback Statistics -> Rating. Is it possible to rate an album more easily, perhaps by adding another button in the top Bar?

have a nice day
Enrico

**Reply by TT-ReBORN** [reply]
I'll see what I can do about custom placeholders but this will be postponed after v3.0 final.
Actually your timing can't be more perfect, I finished something really cool. Check the DEV TESTING thread soon, I'll post it there.
